advocacy_docs/kubernetes/cloud_native_postgresql/e2e.mdx

```diff
@@ -57,7 +57,8 @@ and the following suite of E2E tests are performed on that cluster:
 * Primary endpoint switch in case of switchover in less than 20 seconds;
 * Recover from a degraded state in less than 60 seconds;
 * Physical replica clusters;
-* Storage expansion.
+* Storage expansion;
+* Data corruption;
```

Comment:
@josh-heyer do you think you can fix the final ; with a . after data corruption? not a big deal if not

Reply:
sure